src/features/leverage-tokens/components/leverage-token-mint-modal/index.tsx

```diff
@@ -179,6 +179,9 @@ export function LeverageTokenMintModal({
   // Derive expected tokens from preview data (no local state needed)
   const [transactionHash, setTransactionHash] = useState<`0x${string}` | undefined>(undefined)
   const [error, setError] = useState('')
+  // Store the floor minShares when user reaches confirm step
+  const [savedFloorMinShares, setSavedFloorMinShares] = useState<bigint | undefined>(undefined)
```

Comment:
Is this the right state that we should be introducing?

I think we need just some state for if the quote has been updated. Quote routing/internals could have changed even with the same minShares out. Just want to make sure the user understands that the internals have changed. So they need to confirm again on the confirm step.